tekton/src/pipelines/taskdefs/fvt-apps/common/params-manage-is.yml.j2

```diff
@@ -0,0 +1,24 @@
+- name: mas_instance_id
+  value: $(params.mas_instance_id)
+- name: mas_workspace_id
+  value: $(params.mas_workspace_id)
+- name: devops_cos_crn
+  value: $(params.devops_cos_crn)
+- name: devops_ibmcloud_apikey
+  value: $(params.devops_ibmcloud_apikey)
+- name: fvt_image_registry
+  value: $(params.fvt_image_registry)
+- name: fvt_image_namespace
+  value: fvt-manage
+- name: fvt_image_name
+  value: fvt-ibm-mas-manage
+- name: fvt_image_digest
+  value: $(params.fvt_digest_manage)
+- name: product_channel
+  value: $(params.mas_app_channel_manage)
+- name: product_id
+  value: ibm-mas-manage-is
```

Comment:
`ibm-mas-manage-is` is not a valid product ID (getting a case of de ja vu here, I'm sure already reviewed the same PR a few days ago?)

product ID has to map to the CASE bundle where ultimately the results here will be tied back to.  There is no CASE bundle for industry solutions seperate from Manage, they are all jsut part of manage.

Use a common prefix to group things, e.g. as many ibm-mas suites are prefixed `coreapi-x`, we don't have a ibm-mas-coreapi product ID, we just use `ibm-mas/coreapi-suite1`, `ibm-mas/coreapi-suite2`; same with `ibm-mas/coreidp-suite1` and `ibm-mas/coreidp-suite2`

Reply:
ibm-mas-mobile .. I am expecting all tests under that to be testing the mobile application use of MAS, rather than being manage tests itself.

e.g. if ibm-mas-mobile tests fail it means there's a problem in the mobile application, which isn't part of the ibm-mas-manage CASE bundle, as that's just what is running on OCP.

It's similar to ibm-mas-devops .. it's not a real case bundle, but it's support/ancillary to the operators/CASE bundles.

Reply:
> The names that come after the ibm-mas-manage are not that free as they are keywords used elsewhere in maximo automation framework

You can make the two things independent if needs be .. the names we use can be mapped from nice names to some internal MAF keyword inside our framework (inside the FVT container image before it launches MAF).